haxe/ui/macros/BackendMacros.hx

```diff
@@ -25,7 +25,7 @@ class BackendMacros {
 
     #if macro
 
-    macro private static function loadBackendProperties():Expr {
+    private static function loadBackendProperties():Expr {
```

Comment:
As dicussed, I removed the `macro` because it's unnecessary and causes performance issues.